### DISCUSSION TITLE
Warning: You are using macOS 11.0.

### DISCUSSION THREAD

**Discussion by NOTiFYcouk** [Homebrew/discussions#136] [Everyday usage] [answered]
"We do not provide support for this released but not yet supported version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Discourse, Twitter or any other official channels. You are responsible for
resolving any issues you experience while you are running this
released but not yet supported version."

As of the 13/11/2020 Big Sur 11.x has been release. When is an upgrade expected (to the most useful tool I use) TIA

**Comment by SMillerDev** [comment, marked as answer]
We don't give timelines on features or support as all work is done by volunteers.

**Reply by edbergavera** [reply]
I don't think there would be any major issues so long as you're on Intel-based Macs.

**Reply by Rylan12** [reply]
It's not related to the Intel/ARM transition. We don't have CI available for Big Sur yet so we can't make bottles or do all the testing we need to do which is why it's not officially supported. As Sean mentioned, we don't have a specific timeline but I can assure you that many maintainers are working very hard to get things up and running.

**Reply by zadigre** [reply]
quick question... I'm sorry if this has been answered elsewhere. 
when Big Sur is finally supported, will we have to run special commands or brew update/upgrade will take care of everything?

**Reply by SMillerDev** [reply]
update/upgrade will pull in Big Sur support when it is supported

**Reply by jwfearn** [reply]
I love Homebrew and I am thankful every day for the contributions of the many volunteers who make it possible. And it's fine if Big Sur isn't yet supported, I can wait.

One suggestion though, maybe tone down the wording of the message? Kinda sounds like Homebrew is telling me it's my fault and that I'm somehow making things worse. I'm sure that's not the intention.

**Reply by nertzy** [reply]
@jwfearn Sounds like a great idea for a pull request. I'd encourage you to give it a try.

**Reply by AlbyVA** [reply]
Call me crazy, but isn't there a way we could suppress these messages?

> We do not provide support for this released but not yet supported version.
> You will encounter build failures with some formulae.
> Please create pull requests instead of asking for help on Homebrew's GitHub,
> Twitter or any other official channels.

**Reply by smoothdvd** [reply]
You can switch to master tree to get rid of this message.

**Reply by AlbyVA** [reply]
> You can switch to master tree to get rid of this message.


I'm not sure I understand? How does switching to the master tree address warnings about using Big Sur (macOS 11.0)?

**Reply by SMillerDev** [reply]
The commits that resolves the warnings aren't tested and released yet. But if you're not tracking releases you'll already have them.

**Reply by AlbyVA** [reply]
It looks like I'm only getting this warning when I run
 ( brew upgrade ).

**Comment by NOTiFYcouk** [comment]
Fixed with:

https://apple.stackexchange.com/questions/401899/homebrew-your-clt-does-not-support-macos-11-0

Simplest solution that worked when upgrading to Big Sur. A clean install may not require the solution below. I believe there is some inconsistency with the way CLT is upgraded.

sudo rm -rf /Library/Developer/CommandLineTools
sudo xcode-select --install

**Comment by jakobbg** [comment]
This message no longer appears, and should as such be closed.

**Reply by SMillerDev** [reply]
It's a discussion. Discussions can't be closed, they can only be resolved.

**Reply by seagle0128** [reply]
Why I still get this message? What should I do for it?

**Reply by PerryHuan9** [reply]
Now, I still get this message , and I can't install GNU gcc.

**Reply by gromgit** [reply]
Try `brew update`, then `brew install gcc` again. If that doesn't work, post the output of `brew config` and `brew doctor`.

**Comment by smoothdvd** [comment]
Why there is still this warning?

Homebrew 2.5.11
Homebrew/homebrew-core (git revision e7a9a; last commit 2020-11-27)
Homebrew/homebrew-cask (git revision cbb151; last commit 2020-11-27)

But another machine has no this warning:
Homebrew 2.5.11-271-g79d6c05
Homebrew/homebrew-core (git revision e7a9a; last commit 2020-11-27)
Homebrew/homebrew-cask (git revision cbb151; last commit 2020-11-27)

**Reply by jonchang** [reply]
The versions are different.

**Reply by smoothdvd** [reply]
I run /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)", but still has same result.

**Reply by smoothdvd** [reply]
I know why: one in stable branch, and the other is in master branch.

**Reply by seagle0128** [reply]
Same here.

```
Homebrew 2.5.12
Homebrew/homebrew-core (git revision ff776b; last commit 2020-11-27)
Homebrew/homebrew-cask (git revision 46f063; last commit 2020-11-27)
```

**Reply by Filippo23** [reply]
I am still getting the warning. When will the update be available to the general user?

**Reply by SMillerDev** [reply]
Whenever we publicly release the version that supports Big Sur, which is when we're happy with the support for Big Sur.

**Reply by Filippo23** [reply]
Thank you. I assume that update/upgrade will be all is needed to get the version that supports Big Sur.

**Reply by iMichka** [reply]
Yes, just `brew update/upgrade` regularly and you'll get the update eventually.
